.gitignore

```diff
@@ -1,4 +1,6 @@
 .idea
 node_modules
 react-client-scr/**
-react-native-client-scr/**
\ No newline at end of file
+react-native-client-scr/**
+/packages/cuba-rest-js/dist-browser/
+/packages/cuba-rest-js/dist-node/
```

Comment:
I think it's better to keep such stuff in package-specific .gitignore (`packages/cuba-rest-js/.gitignore`)

Reply:
Fixed.